src/content/docs/more-integrations/open-source-telemetry-integrations/micrometer/micrometer-metrics-registry.mdx

```diff
@@ -1,31 +1,156 @@
 ---
-title: Micrometer metrics registry
-tags:
-  - Integrations
-  - Open source telemetry integrations
-  - Micrometer
+title: Forward Micrometer data to New Relic with OpenTelemetry
 metaDescription: New Relic offers an integration that sends your Micrometer telemetry data to your New Relic account.
 redirects:
-  - /docs/integrations/open-source-telemetry-integrations/micrometer/micrometer-metrics-registry
-  - /docs/integrations/open-source-telemetry-integrations/open-source-telemetry-integration-list/new-relics-micrometer-integration
+    - /docs/integrations/open-source-telemetry-integrations/micrometer/micrometer-metrics-registry
+    - /docs/integrations/open-source-telemetry-integrations/open-source-telemetry-integration-list/new-relics-micrometer-integration
 freshnessValidatedDate: never
 ---
 
-[Micrometer](https://micrometer.io/) provides a simple facade over the instrumentation clients for the most popular monitoring systems, allowing you to instrument your JVM-based application code without vendor lock-in.
+import opentelemetryMicrometerWithOtel from 'images/opentelemetry_screenshot-full_micrometer-with-otel.webp'
 
-New Relic's Micrometer metrics registry sends your Micrometer telemetry data to your New Relic account. Don't have one? [Sign up for free](https://newrelic.com/signup). No credit card required.
+New Relic supports [Micrometer data](https://micrometer.io/) so you can view all of your observability metrics in one platform. You'll configure the [OpenTelemetry Micrometer bridge](https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation/micrometer/micrometer-1.5/library) with the OpenTelemetry SDK, then use OpenTelemetry Protocol (OTLP) to forward your Micrometer data to New Relic.
 
-Resources for our Micrometer integration:
+<img
+    title="Micrometer bridged with OpenTelemetry in New Relic"
+    alt="A screenshot of the Micrometer summary page when instrumented with OpenTelemetry"
+    src={opentelemetryMicrometerWithOtel}
+/>
 
-* [README](https://github.com/newrelic/micrometer-registry-newrelic)
-* [Integration specs](https://github.com/newrelic/newrelic-exporter-specs/tree/master/micrometer)
+<figcaption>
+Go to **[one.newrelic.com](https://one.newrelic.com) > All Capabilities > APM & Services**, then find the **Services - OpenTelemetry** section: View your Micrometer data in New Relic when bridged through OpenTelemetry
+</figcaption>
 
-<InstallFeedback />
+## Compatibility and requirements [#requirements]
 
-## Find your data
+Before working through these procedures, you should:
```

Comment:
This isn't quite right. The procedure below tells you how to install the OpenTelemetry micrometer bridge and how to configure the OpenTelemetry metrics SDK to export to New Relic via OTLP. This whole Compatibility and requirements section can probably be deleted, unless you want to talk about  requirement for a license key or something.